Theme.html

```diff
@@ -4,7 +4,8 @@
 		<title>Theme JSON</title>
 		<link rel="stylesheet" href="https://static.tumblr.com/2wliono/oG8ps6bg4/jsondrop.css">
 		<link rel="stylesheet" href="https://static.tumblr.com/2wliono/PXMps6cfk/colour.css">
-		<script src="https://static.tumblr.com/2wliono/jGHps6w0v/jsondrop.js"></script>
+		<script src="https://static.tumblr.com/2wliono/I75ps707w/jsondrop.js"></script>
+		<meta name="text:Auto-Expand Depth" content="2"/>
```

Comment:
can't get this work, looks like tumblr theme variable names cant have hyphens :(
`"text:AutoExpand Depth"` works fine though